std_msgs/Header header
hololens_project/Obstacle[] obstacles
